internal/client/apps/keys.go

```diff
@@ -62,9 +61,8 @@ func CreateKey(developerEmail string, appID string, consumerKey string, consumer
 
 	// since the API does not support adding products when creating a key, use a second API call to add products
 	if len(apiProducts) > 0 {
-		apiclient.ClientPrintHttpResponse.Set(false)
-		respBody, err = UpdateKeyProducts(developerEmail, appID, consumerKey, apiProducts)
-		apiclient.ClientPrintHttpResponse.Set(apiclient.GetCmdPrintHttpResponseSetting())
+		apiclient.ClientPrintHttpResponse.Set(true)
```

Comment:
Instead of true, please replace with `apiclient.ClientPrintHttpResponse.Set(apiclient.GetCmdPrintHttpResponseSetting())`